### PR TITLE
Add more test on RPKI signed data

### PR DESCRIPTION
**Pull request by wibisono**
Add test highlighting and documenting the main difference of RPKISignedDataGenerator and original CMSSignedDataGenerator

### CODE REVIEW

File: src/test/java/net/ripe/rpki/commons/crypto/cms/RPKISignedDataGeneratorTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import org.junit.Before;
+import org.junit.Test;
+
+import java.security.cert.X509Extension;
+import java.util.Collections;
+import java.util.List;
+
+public class RPKISignedDataGeneratorTest {
+
+    final RPKISignedDataGenerator rpkiCmsSubject = new RPKISignedDataGenerator();
```
Comment (ties) [inline src/test/java/net/ripe/rpki/commons/crypto/cms/RPKISignedDataGeneratorTest.java:51]: A nit that about the core of this pr: There are some todos in the generator itself. Can we check if those are still relevant?

If so, I think those describe the edge cases that we should check in the test.

**Review by ties — APPROVED** [review]

File: src/test/java/net/ripe/rpki/commons/crypto/cms/RPKISignedDataGeneratorTest.java (hunk 1 above)
Comment (wibisono) [reply src/test/java/net/ripe/rpki/commons/crypto/cms/RPKISignedDataGeneratorTest.java:51]: Indeed, those TODOs are inherited from original CMS generator that I don't really get. Will take some time to digest it.
